### ISSUE TITLE
Strava and Runkeeper Sync

### ISSUE THREAD

**Issue by nycjoker** [cpfair/tapiriik#209]
I've been using tapiriik for sometime to sync my runs from Strava to Runkeeper. I've been using the same device as well, a GPS watch with HR. It has worked perfectly until recently. 

When the data is uploaded to Runkeeper the map has frequent pauses displayed. I do not pause my runs. On Strava these runs are displayed perfectly fine. As a test, I exported the TCX file from Strava and manually uploaded to Runkeeper. Data uploaded and the run appeared  correctly with no pauses. As I continue to use tapiriik my Runkeeper sync still generates/adds the pauses. 

Anyone know what changed and why this is happening?

**Comment by cmnyc** [comment]
Have you checked the "Similate auto pause" option,
on tapiriik, for Runkeeper?

**Comment by nycjoker** [comment]
Yes. After this started happening I checked the configuration settings on tapiriik. The checkbox for "Simulate auto pause" in Runkeeper is not selected.

**Comment by EndeNeu** [comment]
Same thing happened to me just today, a 40 min 6.5km activity has been reduced to 2 km in 2 seconds when imported from Runkeeper to Strava, I deleted the activity and manually exported the activity from Runkeeper in GPX format and import it and it report the correct distance but the wrong time (4 seconds), I'm guessing this has to do with one of both service rather than tapiriik.
